CONTRIBUTING.md

```diff
@@ -86,6 +86,63 @@ You can then fill in the `term`, `def`, and (if appropriate) `acronym` for that
       toutes les fonctions.
 ```
 
+## 3. Adding a new language
+
+If you want to add a term or define a term in a new language, you can quickly check if your language is defined in glosario by heading to `https://glosario.carpentries.org/[lang]`, replacing `[lang]` with the [two-letter ISO 639 language code][iso639-table-en]. If you see a 
```

Comment:
Would it be easier to point to https://glosario.carpentries.org/ and check on the list of languages on the right?

Reply:
Looking at the home page definitely works, but it will eventually get frustrating when the number of languages grows. I'm okay with either one.

Reply:
good point, let's keep the way you wrote it.